### PR TITLE
fix: new raster ingestion modifications (MAPCO-4302)

### PR DESCRIPTION
**Pull request by RonitKissis**
updated raster ingestion types for polygon parts.

### CODE REVIEW

File: src/yaml/ingestionTrigger/rasterLayerExample.yaml [hunk 1]
```diff
@@ -7,20 +7,21 @@ components:
     NewRasterLayerExample:
       value:
         metadata:
-          productId: string
+          productId: c5e3f820-b2bd-4f0b-a70f-c98bf33b2692
```
Comment (asafMasa) [inline src/yaml/ingestionTrigger/rasterLayerExample.yaml:10]: revert

File: src/yaml/ingestionTrigger/rasterLayerExample.yaml [hunk 2]
```diff
@@ -7,20 +7,21 @@ components:
...
           transparency: TRANSPARENT
-          region: string
+          region: 
+            - string
           classification: '6'
           producerName: string
           scale: 100000000
         partData:
-          - id: string
+          - id: c5e3f820-b2bd-4f0b-a70f-c98bf33b2692
```
Comment (asafMasa) [inline src/yaml/ingestionTrigger/rasterLayerExample.yaml:24]: revert

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

**Review by netanelC — APPROVED** [review]
Not related to 3D

**Review by alebinson — APPROVED** [review]
Basically looks good. That mean ONE thing: INTEGRATION will be LONGER

**Pull request merged**